src/main/java/com/puppycrawl/tools/checkstyle/checks/annotation/AnnotationLocationCheck.java

```diff
@@ -268,27 +268,27 @@ public void visitToken(DetailAST ast) {
     }
 
     /**
-     * Some javadoc.
-     * @param modifierNode Some javadoc.
-     * @return Some javadoc.
+     * Checks whether a given modifier node has an annotation.
+     * @param modifierNode modifier node.
+     * @return true if a given modifier node has an annotation.
      */
     private static boolean hasAnnotations(DetailAST modifierNode) {
         return modifierNode != null && modifierNode.findFirstToken(TokenTypes.ANNOTATION) != null;
     }
 
     /**
-     * Some javadoc.
-     * @param modifierNode Some javadoc.
-     * @return Some javadoc.
+     * Returns an annotation level.
```

Comment:
@rnveach 
Maybe it is better to create an issue and perform a small refactoring of the code, because annotation level is not a correct term. The term is present in methods names and the check logic.